### ISSUE TITLE
Regression error in parser with continuation line handling in PIC-clause

### ISSUE THREAD

**Issue by Reinhard-Prehofer** [uwol/cobol85parser#57]
This error happend after I applied the display '---' patch.
I am rather sure the single DOT in the continuation line causes the problem. Just the line where I appended the comment underneath

```
        10 FILLER PIC X(99)          VALUE
        '00010019EG101_1       048EG;Ges;1;2;2+;3;3+;4;4+;5;5+;6;6+;4a;4
      -  'a+;4b;4b+;'.
              10 FILLER PIC X(99)          VALUE '00000043EG107_2_1_FH  12.05.16
002580-          '034Alter;Ges;Erf;N-Erf;Ges;Erf;N-Erf;'.               
002590                                                                  
002600        10 FILLER PIC X(99)          VALUE '00000019EG108_1       12.05.16
002610-          '050EG;Entg;Üb;Nacht;Sonn;Feier;Sam;Bereit;Ruf;WS;Sch;'
002620-                                                                .
      * YES! There is just a single DOT In above line !
002630        10 FILLER PIC X(99)          VALUE '00000019EG108_2       12.05.16
002640-          '052EG;HW;Erschw;KGZ;EGZ;MTP;VergZ;KOZ;ATZ1;ATZ2;SV;VBL
002650-          ';'.                                                   
002660                
```
Parsing error is as follows, file is 
[a2600TDL.CBL.txt](https://github.com/uwol/cobol85parser/files/1581984/a2600TDL.CBL.txt)
attached:

`File a2600TDL.CBL threw error syntax error in line 23:17 extraneous input 'FILLER' expecting {ABORT, ALIGNED, ALL, AS, ASCII, ASSOCIATED_DATA, ASSOCIATED_DATA_LENGTH, ATTRIBUTE, AUTO, AUTO_SKIP, BACKGROUND_COLOR, BACKGROUND_COLOUR, BEEP, BELL, BINARY, BIT, BLANK, BLINK, BOUNDS, BY, CAPABLE, CCSVERSION, CHANGED, CHANNEL, CLOSE_DISPOSITION, COBOL, COMMITMENT, COMMON, COMP, COMP_1, COMP_2, COMP_3, COMP_4, COMP_5, COMPUTATIONAL, COMPUTATIONAL_1, COMPUTATIONAL_2, COMPUTATIONAL_3, COMPUTATIONAL_4, COMPUTATIONAL_5, CONTENT, CONTROL_POINT, CONVENTION, CRUNCH, CURSOR, DATE, DEFAULT, DEFAULT_DISPLAY, DEFINITION, DFHRESP, DFHVALUE, DISK, DISPLAY, DISPLAY_1, DONTCARE, DOUBLE, EBCDIC, EMPTY_CHECK, ENTER, ENTRY_PROCEDURE, ERASE, EOL, EOS, ESCAPE, EVENT, EXCLUSIVE, EXPORT, EXTENDED, EXTERNAL, FALSE, FOREGROUND_COLOR, FOREGROUND_COLOUR, FULL, FUNCTIONNAME, FUNCTION_POINTER, GLOBAL, GRID, HIGHLIGHT, HIGH_VALUE, HIGH_VALUES, IMPLICIT, IMPORT, INDEX, INTEGER, IS, JUST, JUSTIFIED, KANJI, KEPT, KEYBOARD, LANGUAGE, LB, LD, LEADING, LEFTLINE, LENGTH_CHECK, LIBACCESS, LIBPARAMETER, LIBRARY, LIST, LOCAL, LOCK, LONG_DATE, LONG_TIME, LOWER, LOWLIGHT, LOW_VALUE, LOW_VALUES, MMDDYYYY, NAMED, NATIONAL, NATIONAL_EDITED, NETWORK, NO_ECHO, NULL, NULLS, NUMERIC_DATE, NUMERIC_TIME, OCCURS, ODT, ORDERLY, OVERLINE, OWN, PACKED_DECIMAL, PASSWORD, PIC, PICTURE, POINTER, PORT, PRINTER, PRIVATE, PROCEDURE_POINTER, PROCESS, PROGRAM, PROMPT, QUOTE, QUOTES, READER, REMOTE, REAL, RECEIVED, RECORD, RECURSIVE, REDEFINES, REF, REFERENCE, REMOVE, REQUIRED, REVERSE_VIDEO, SAVE, SECURE, SHARED, SHAREDBYALL, SHAREDBYRUNUNIT, SHARING, SHORT_DATE, SIGN, SPACE, SPACES, STRING, SYMBOL, SYNC, SYNCHRONIZED, TASK, THREAD, THREAD_LOCAL, TIMER, TODAYS_DATE, TODAYS_NAME, TRAILING, TRUE, TRUNCATED, TYPE, TYPEDEF, UNDERLINE, USAGE, USING, VALUE, VALUES, VIRTUAL, WAIT, WITH, YEAR, YYYYMMDD, YYYYDDD, ZERO, ZERO_FILL, ZEROS, ZEROES, DOT_FS, NONNUMERICLITERAL, '66', '77', '88', INTEGERLITERAL, NUMERICLITERAL, IDENTIFIER}`

**Comment by uwol** [comment]
Ok, fixed in 709148b86532f0c5ca77a74994d6e5096b6d0de6.

Strictly speaking, it think this example is non-compliant to the specification, as a `'` in column 72 of the continued line has to be followed by a double `''` at the beginning of the continuation line.  

However, of course we strive for maximal compatibility. As the number of line continuation rules is growing steadily, probably I'll have to refactor them in a later release version. I added unit tests for all of these line continuation rules, so that regressions hopefully will be minimized.

Thanks!

**Comment by Reinhard-Prehofer** [comment]
(Y) - If I look at the customer's coding here, compliant or not, I even got the impression that they have written the value declaration from RIGHT to left!
best regards!
